### ISSUE TITLE
Quiz | Question Progress panel contrast

### ISSUE THREAD

**Issue by msoupcoff** [DukeLearningInnovation/duke-sakai-skins#250]
Within the question progress panel, the part number sections fails color contrast when hovering over.

![QuizProgressPanelPartHover](https://user-images.githubusercontent.com/1220785/115603841-a4556080-a2ae-11eb-9cf0-52620ea12a0c.gif)
